tests/typecheck/contrib/messages/test_views.yml

```diff
@@ -0,0 +1,15 @@
+-   case: view_valid_with_custom_form
+    main: |
+      from django import forms
+      from django.contrib.messages.views import SuccessMessageMixin
+      from django.forms import ModelForm
+      from django.http.response import HttpResponse
+
+
+      class MyForm(forms.Form):
+          pass
+
+
+      class Test(SuccessMessageMixin[MyForm], ModelForm):
+          def form_valid(self, form: MyForm) -> HttpResponse:
```

Comment:
Thanks, but we don't add tests covering ordinary stubs (except for particularly complex cases). Covering stubs with tests would require more test code than actual type stub code :)